### PR TITLE
Update configuration for 6.0 release

### PR DESCRIPTION
**Pull request by cirocosta**
### Why do we need this PR?

With the 6.0 release, some flags were added, other removed.

https://ci.concourse-ci.org/teams/main/pipelines/concourse/jobs/k8s-check-helm-params/builds/65

This PR makes sure we get that right :grin: 

### Changes proposed in this pull request

- remove `noop`
- add `job-scheduling-max-in-flight`
- split maxconns in api and backend
- remove enableLidar
- add `useContainerd`

- ~[ ] Variables are documented in the `README.md`~ the vars are all concourse specific

### CODE REVIEW

**Comment by cirocosta** [comment]
ps: w/ regards to TSA_TEAM_AUTHORIZED_KEYS_FILE, see https://github.com/concourse/ci/pull/262

**Comment by cirocosta** [comment]
Hey,

I just rebased and added `useContainerd` to the worker's garden section (the same way one can use `useHoudini`).

thanks!

File: templates/web-deployment.yaml [hunk 1]
```diff
@@ -63,17 +63,21 @@ spec:
...
+            {{- end }}
+            {{- if .Values.concourse.web.backendMaxConns }}
+            - name: CONCOURSE_BACKEND_MAX_CONNS
+              value: {{ .Values.concourse.web.backendMaxConns | quote }}
             {{- end }}
             {{- if .Values.concourse.web.enableGlobalResources }}
             - name: CONCOURSE_ENABLE_GLOBAL_RESOURCES
               value: {{ .Values.concourse.web.enableGlobalResources | quote }}
             {{- end }}
-            {{- if .Values.concourse.web.enableLidar }}
```
Comment (cirocosta) [inline templates/web-deployment.yaml]: some flags like this were deprecated - should I include something about it in CHANGELOG.md?

Comment (pivotal-jamie-klassen) [reply templates/web-deployment.yaml]: Probably wouldn't hurt - we'll try to be thorough with the GitHub release notes, but chart consumers might forget to read those.

File: values.yaml [hunk 2]
```diff
@@ -1311,9 +1311,12 @@ concourse:
     externalGardenUrl:
 
     garden:
-      ## Path to the 'gdn' executable (or leave as 'gdn' to find it in $PATH)
+      ## Path to the executable to run the garden backend ('gdn' or
+      ## 'containerd').
       ##
-      bin: gdn
+      ## If not specified, the right binary will be search in $PATH.
```
Comment (pivotal-jamie-klassen) [inline values.yaml]:
```suggestion
      ## If not specified, Concourse will search $PATH for the right binary.
```
flows a little bit better

File: templates/web-deployment.yaml (hunk 1 above)
Comment (cirocosta) [reply templates/web-deployment.yaml]: @pivotal-jamie-klassen, I just submitted https://github.com/concourse/concourse-chart/pull/59/commits/10e19b205ae512dd9d327cc76331678a885a529c to update the changelog 👍

**Comment by vito** [comment]
@cirocosta Thanks for putting this together! I think it probably just needs some touching up now, plus removing the Riemann properties to make [the build](https://ci.concourse-ci.org/teams/main/pipelines/concourse/jobs/k8s-check-helm-params/builds/132) happy.

**Comment by cirocosta** [comment]
doone @vito

**Comment by vito** [comment]
@cirocosta actually, do we need to bump the version?

**Comment by cirocosta** [comment]
oh damn, we definitely do - as well as having a little changelog note written given the breaking changes on the `values.yaml`. will do tomorrow :+1:

File: Chart.yaml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: concourse
-version: 9.1.2
-appVersion: 5.8.0
+version: 10.0.0
```
Comment (cirocosta) [inline Chart.yaml]: :eyes:

**Pull request merged**